### DISCUSSION TITLE
My camera shows green screen but android app works

### DISCUSSION THREAD

**Discussion by primski** [blakeblackshear/frigate#4054] [Ask A Question] [answered]
I have a chinese clone crap; and RTSP feed seems to hang every now and then ( every 5 minutes ) ... frigate shows green screen instead of picture, as if it can't connect, but android ( icSee) works, i don't understand how ... ( do android apps for cameras use different protocol ?)

also, how do i passwrod protect frigate? it's publicly accessible and no authentication support?

thirdly, every time camera detects something, it hands and frigate need to be rebooted

i probably have more but forgot for now ...

thanks. lovely software though, just needs a few more features to make it usable 100% of the time ..

**Comment by NickM-27** [comment]
> I have a chinese clone crap; and RTSP feed seems to hang every now and then ( every 5 minutes ) ... frigate shows green screen instead of picture, as if it can't connect, but android ( icSee) works, i don't understand how ... ( do android apps for cameras use different protocol ?)

To help would need to see frigate logs, 

> also, how do i passwrod protect frigate? it's publicly accessible and no authentication support?

You can use any number of options for authentication including nginx proxy manager, oauth2proxy, traefik, etc. There's no need

> thirdly, every time camera detects something, it hands and frigate need to be rebooted

need to see logs

**Comment by primski** [comment]
I found this, maybe gives a hint?

> 
> frigate-security | Fatal Python error: Bus error
> frigate-security |
> frigate-security | Thread 0x00007f320e3d0700 (most recent call first):
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 312 in wait
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/queues.py", line 233 in _feed
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 892 in run
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 954 in _bootstrap_inner
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 912 in _bootstrap
> frigate-security |
> frigate-security | Current thread 0x00007f3210bd1700 (most recent call first):
> frigate-security |   File "/opt/frigate/frigate/video.py", line 168 in capture_frames
> frigate-security |   File "/opt/frigate/frigate/video.py", line 306 in run
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 954 in _bootstrap_inner
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 912 in _bootstrap
> frigate-security |
> frigate-security | Thread 0x00007f32133d2700 (most recent call first):
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/synchronize.py", line 261 in wait
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/synchronize.py", line 349 in wait
> frigate-security |   File "/opt/frigate/frigate/video.py", line 232 in run
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 954 in _bootstrap_inner
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 912 in _bootstrap
> frigate-security |
> frigate-security | Thread 0x00007f3219bd3700 (most recent call first):
> frigate-security |   File "/opt/frigate/frigate/log.py", line 64 in run
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 954 in _bootstrap_inner
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 912 in _bootstrap
> frigate-security |
> frigate-security | Thread 0x00007f321feda740 (most recent call first):
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 1049 in _wait_for_tstate_lock
> frigate-security |   File "/usr/lib/python3.9/threading.py", line 1033 in join
> frigate-security |   File "/opt/frigate/frigate/video.py", line 337 in capture_camera
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/process.py", line 108 in run
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/process.py", line 315 in _bootstrap
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 71 in _launch
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 19 in __init__
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/context.py", line 277 in _Popen
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/context.py", line 224 in _Popen
> frigate-security |   File "/usr/lib/python3.9/multiprocessing/process.py", line 121 in start
> frigate-security |   File "/opt/frigate/frigate/app.py", line 286 in start_camera_capture_processes
> frigate-security |   File "/opt/frigate/frigate/app.py", line 364 in start
> frigate-security |   File "/opt/frigate/frigate/__main__.py", line 16 in <module>
> frigate-security |   File "/usr/lib/python3.9/runpy.py", line 87 in _run_code
> frigate-security |   File "/usr/lib/python3.9/runpy.py", line 197 in _run_module_as_main
> 

thanks for auth tips, i'll research the options

**Reply by NickM-27** [reply]
That error is covered in the docs: https://docs.frigate.video/faqs#fatal-python-error-bus-error

**Comment by primski** [comment, marked as answer]
alright, i've increased the shm size for this container, will see... many thanks.
